packages/reactive-element/src/decorators/query-all.ts

```diff
@@ -59,13 +59,25 @@ let fragment: DocumentFragment;
  * @category Decorator
  */
 export function queryAll(selector: string): QueryAllDecorator {
-  return (() => {
-    return {
+  return ((
+    obj: object,
+    name: PropertyKey | ClassAccessorDecoratorContext<unknown, unknown>
```

Comment:
I think this:

```suggestion
    nameOrContext: PropertyKey | ClassAccessorDecoratorContext<unknown, unknown>
```

will make the `typeof name !== 'object'` below a lot more clear.

Reply:
Less necessary now that we're just passing this along